### PR TITLE
Added tutorial to install SCT with WSL for Windows users

### PR DESCRIPTION
**Pull request by lrouhier**
This PR aims at changing the Readme to provide links to install via docker as reported in the wiki and to replace the VM install with the windows subsystem for linux one.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,32 +94,15 @@ git pull
...
-cd /home/brain
-wget https://download.virtualbox.org/virtualbox/6.0.4/VBoxGuestAdditions_6.0.4.iso
-mkdir disk_VM
-sudo mount VBoxGuestAdditions_6.0.4.iso disk_VM/
-sudo disk_VM/VBoxLinuxAdditions.run
-sudo reboot
-~~~
-
-After the VM has restarted, you can install SCT!
+In order to install sct on Windows 10, it is possible to install it on windows subsystem for linux (WSL).
```
Comment (jcohenadad) [inline README.md]: i would remove this sentence-- it is redundant with the title
```suggestion
```

**Comment by jcohenadad** [comment]
@lrouhier any update on this?

**Comment by lrouhier** [comment]
Sorry it feel through the crack. I just checked it out I should be able to ask for a review in an hour or so (just confirming something with the install process). It still needs a few modifications. I am on it.

**Comment by lrouhier** [comment]
I think its good. I modify the order because I suspect most users having 'unsupported OS' are windows users.

File: README.md [hunk 2]
```diff
@@ -96,35 +96,21 @@ At anytime, you can update your local SCT installation to the latest version on
 git pull
 ~~~
 
-### Installation with Docker
 
-In order to install the SCT in Windows or other unsupported OS, we provide a solution via Docker. This solution will allow you to use all the resources of SCT including the programs with graphical interface. In addition to the SCT, the Docker image contains the FSLeyes program for the visualization of the NIFTI volumes that you process.
+### Installation with Windows Subsystem for Linux (Windows 10).
 
-[Tutorial how to install SCT via Docker](https://github.com/neuropoly/sct_docker#spinal-cord-toolbox-docker)
+Windows subsystem for Linux (WSL) is a new Windows 10 feature that enables you to run native Linux command-line tools directly on Windows. That way, it is possible to install a Linux version SCT and use it within your Windows 10 environment.
```
Comment (jcohenadad) [inline README.md:102]:
```suggestion
Windows subsystem for Linux (WSL) is available on Windows 10 and it makes it possible to run native Linux programs. 
```

File: README.md [hunk 3]
```diff
@@ -96,35 +96,21 @@ At anytime, you can update your local SCT installation to the latest version on
...
 
-### Installation in Virtual Machine (VM)
+[Tutorial how to install SCT on WSL](https://github.com/neuropoly/spinalcordtoolbox/wiki/SCT-on-Windows-10:-Installation-instruction-for-SCT-on-Windows-subsytem-for-linux)
```
Comment (jcohenadad) [inline README.md:104]:
```suggestion
[Tutorial to install SCT on WSL](https://github.com/neuropoly/spinalcordtoolbox/wiki/SCT-on-Windows-10:-Installation-instruction-for-SCT-on-Windows-subsytem-for-linux)
```

File: README.md [hunk 4]
```diff
@@ -96,35 +96,21 @@ At anytime, you can update your local SCT installation to the latest version on
...
-Then, you need to install driver updates for the video card. Start the VM, open a Terminal and run:
-~~~
-cd /home/brain
-wget https://download.virtualbox.org/virtualbox/6.0.4/VBoxGuestAdditions_6.0.4.iso
-mkdir disk_VM
-sudo mount VBoxGuestAdditions_6.0.4.iso disk_VM/
-sudo disk_VM/VBoxLinuxAdditions.run
-sudo reboot
-~~~
+In order to install the SCT unsupported OS, we provide a solution via Docker. This solution will allow you to use all the resources of SCT including the programs with graphical interface. In addition to the SCT, the Docker image contains the FSLeyes program for the visualization of the NIFTI volumes that you process.
```
Comment (jcohenadad) [inline README.md:109]:
```suggestion
If none of the above solution work, SCT can be installed via Docker. 
```

File: README.md [hunk 5]
```diff
@@ -96,35 +96,21 @@ At anytime, you can update your local SCT installation to the latest version on
...
+
+[Tutorial how to install SCT via Docker](https://github.com/neuropoly/spinalcordtoolbox/wiki/testing#run-docker-image)
```
Comment (jcohenadad) [inline README.md:111]:
```suggestion
[Tutorial to install SCT via Docker](https://github.com/neuropoly/spinalcordtoolbox/wiki/testing#run-docker-image)
```

File: README.md [hunk 6]
```diff
@@ -96,35 +96,21 @@ At anytime, you can update your local SCT installation to the latest version on
...
 
-After the VM has restarted, you can install SCT!
+[Tutorial How to use GUI with the docker image](https://github.com/neuropoly/spinalcordtoolbox/wiki/testing#run-docker-with-gui)
```
Comment (jcohenadad) [inline README.md:113]: included in the link above
```suggestion
```

**Comment by lrouhier** [comment]
I think it's good if you want to merge.

**Pull request merged**